.github/workflows/ci.yml

```diff
@@ -20,17 +27,28 @@ jobs:
           npx commitlint --version
       - name: Run tests to validate our plugins
         run: yarn jest
-
   sanitycheck:
     runs-on: ubuntu-20.04
+    container:
+      image: "ubuntu:20.04"
     steps:
+      - name: Install required dependencies
+        run: |
+          apt-get update && apt-get install -y sudo
+          sudo apt install -y git
+          sudo apt-get install -y gcc g++ make
+          
+          sudo apt install -y curl
+          curl -sL https://deb.nodesource.com/setup_18.x | sudo -E bash -
+          sudo apt-get install -y nodejs
       - uses: actions/checkout@v2
         with:
           submodules: recursive
           # needed because of commit-lint, see https://github.com/conventional-changelog/commitlint/issues/3376
           fetch-depth: 0
-      - name: Install dependencies of commitlint
-        run: sudo apt install --yes npm && npm install @commitlint/config-conventional
+      - name: ownership workaround
```

Comment:
@parhamsaremi in geewallet there's a comment that has a URL directing to the bug; if you don't include it, the next developer will think "workaround to what issue???"

Reply:
right I guess I missed that :+1: